loader/loader_windows.c

```diff
@@ -1146,6 +1146,7 @@ VkResult windows_get_loader_settings_file_path(const struct loader_instance *ins
 
         rtn_value = RegOpenKeyEx(HKEY_LOCAL_MACHINE, VK_SETTINGS_INFO_REGISTRY_LOC, 0, access_flags, &key);
         if (ERROR_SUCCESS != rtn_value) {
+            result = VK_ERROR_FEATURE_NOT_PRESENT;
```

Comment:
This should also be applied to line 1133 so that both regular security and high security have the fix.

Reply:
Yes, done